src/main/java/sparksoniq/jsoniq/item/ObjectItem.java

```diff
@@ -77,11 +77,7 @@ else if (values.size() == 1) {
                 valueList.add(value);
             }
```

Comment:
This is not related to the changes you've made but could you remove the @param itemMetadata from JavaDoc in line 59